extensions/cornerstone/src/contextProviders/ViewportActionCornersProvider.tsx

```diff
@@ -0,0 +1,114 @@
+import React, {
+  ReactNode,
+  createContext,
+  useCallback,
+  useContext,
+  useEffect,
+  useReducer,
+  useState,
+} from 'react';
+import PropTypes from 'prop-types';
+
+import { ViewportActionCornersLocations } from '@ohif/ui';
+import ViewportActionCornersService, {
+  VIEWPORT_ACTION_ARROWS_COMPONENT_ID,
+  VIEWPORT_STATUS_COMPONENT_ID,
+} from '../services/ViewportActionCornersService/ViewportActionCornersService';
+
+const DEFAULT_STATE = {
+  default: {
+    actionComponents: {},
+    cornerComponents: {
+      [ViewportActionCornersLocations.topLeft]: [],
+      [ViewportActionCornersLocations.topRight]: [],
+      [ViewportActionCornersLocations.bottomLeft]: [],
+      [ViewportActionCornersLocations.bottomRight]: [],
+    },
+  },
```

Comment:
I'm a little bit confused, I didn't expect the actionComponents to exist, and only we have this 

```js
const DEFAULT_STATE = {
  default: {
    [ViewportActionCornersLocations.topLeft]: [],
    [ViewportActionCornersLocations.topRight]: [],
    [ViewportActionCornersLocations.bottomLeft]: [],
    [ViewportActionCornersLocations.bottomRight]: [],
  },
}
```

am i missing something?